instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/ui/routeline/RouteLineLayersTest.kt

```diff
@@ -54,6 +55,7 @@ class RouteLineLayersTest : BaseTest<BasicNavigationViewActivity>(
     }
 
     @Test
+    @UiThreadTest
```

Comment:
All interactions with `MapboxNavigation` are required to be done from the main thread. It's just that only in some cases the exception is thrown. Shouldn't we approach this task systematically? For example. run all tests on main thread if it's possible. We use `MapboxNavigation` pretty much everywhere, who knows which tests will suddenly start failing tomorrow or in a month.